challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+  input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
   #implement your solution here
+  input.reject{|e| e != "dog"}
 end
 
 def no_dogs_allowed(input)
   #implement your solution here
+  input.reject{|e| e== "dog"}
```

Comment:
You've already used reject in your fetch_the_dog method. Can you find another enumerable method to solve this?